.env.template

```diff
@@ -10,4 +10,9 @@ REACT_APP_config_path = /opt/rucio/lib/rucio/web/ui/common/saml/
 
 [oidc]
 REACT_APP_idpsecrets = /opt/rucio/etc/idpsecrets.json
-REACT_APP_admin_issuer = wlcg
\ No newline at end of file
+REACT_APP_admin_issuer = wlcg
+
+[login_page_org_images]
+REACT_APP_login_page_org_image_1 = ./images/favicon.png
```

Comment:
Let's say that the Rucio logo is always going to be present. In addition, Let's just use 2 images, `REACT_APP_login_page_image_primary` and `REACT_APP_login_page_image_secondary`.
We can render all images in the order ( L-R) in a single row : primary -> rucio_logo -> secondary
@bari12 @cserf @mlassnig any comments?

Reply:
Sounds good to me, made the changes!